Header header
float64[] actualTCPPose
float64[] targetTCPPose
float64[] actualTCPSpeed
float64[] targetTCPSpeed
float64[] actualTCPForce
float64[] actualToolAccel
int64 digitalInputBits
